packages/react-components/react-utilities/src/utils/properties.ts

```diff
@@ -126,12 +126,25 @@ export const baseElementProperties = toObjectMap([
   'name', // global
 ]);
 
+/**
+ * An array of microdata attributes that are allowed on every html element type.
+ *
+ * @public
+ */
+export const microdataProperties = toObjectMap([
```

Comment:
nit: would be nice to add a link to MDN about microdata and its attributes
https://developer.mozilla.org/en-US/docs/Web/HTML/Microdata

Reply:
Good idea. Too bad I didn't see this before the hitting the merge button hard 🙈